openstudiocore/src/energyplus/Test/EMS_GTest.cpp

```diff
@@ -1058,16 +1059,16 @@ TEST_F(EnergyPlusFixture, ForwardTranslatorGlobalVariable_EMS) {
   Model model;
 
   // add global variable
-  EnergyManagementSystemGlobalVariable var(model, "glob var");
```

Comment:
@jmarrec does `var(model, "glob var")` throw?  If so we should add an expect throw test

Reply:
And I usually check that the number of objects did not increase when a ctor throws

Reply:
we do test that here but it could be expanded
https://github.com/NREL/OpenStudio/blob/develop/openstudiocore/src/model/test/EnergyManagementSystemGlobalVariable_GTest.cpp#L55